lib/data-types.js

```diff
@@ -564,6 +661,9 @@ var ENUM = function(value) {
 util.inherits(ENUM, ABSTRACT);
 
 ENUM.prototype.key = ENUM.key = 'ENUM';
+ENUM.prototype.validate = function(value) {
+  return true;
```

Comment:
Hmm, maybe check that the value is in the allowed ones? Not sure you have access to those here though..